source/MRSymbolMesh/MRAlignTextToMesh.h

```diff
@@ -34,4 +35,27 @@ struct TextMeshAlignParams : SymbolMeshParams
 
 // Creates symbol mesh and aligns it to given surface
 MRSYMBOLMESH_API Expected<Mesh> alignTextToMesh( const Mesh& mesh, const TextMeshAlignParams& params );
-}
+
+struct CurvedAlignTextToPathParams : SymbolMeshParams
+{
+    /// Relative position of curve line (y=pivotY) in contours bounding box:
+    /// 0 - bottom, 0.5 - center, 1 - top
+    float pivotY = 0;
```

Comment:
should we have Vector2f pivot?

Reply:
only `y` is fixed in the parameters, and `x` moves along the curve